frontend/src/index.html

```diff
@@ -15,6 +15,9 @@
 
 <app-content></app-content>
 
+<script src="https://maps.googleapis.com/maps/api/js?sensor=false"></script>
```

Comment:
@VitaNuova You can remove this import, as right know there are 2 of them in our index.html